src/modules/azure.rs

```diff
@@ -0,0 +1,267 @@
+use std::fs::File;
+use std::io::{BufReader, Read};
+use std::path::{Path, PathBuf};
+
+use super::{Context, Module, RootModuleConfig};
+
+type JValue = serde_json::Value;
+
+use crate::configs::azure::AzureConfig;
+use crate::formatter::StringFormatter;
+
+type SubscriptionName = String;
+
+pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
+    let mut module = context.new_module("azure");
+    let config = AzureConfig::try_load(module.config);
+
+    if config.disabled {
+        return None;
+    };
+
+    let subscription_name: Option<SubscriptionName> = get_azure_subscription_name(context);
+    if subscription_name.is_none() {
+        log::info!("Could not find Azure subscription name");
+        return None;
+    };
+
+    let parsed = StringFormatter::new(config.format).and_then(|formatter| {
+        formatter
+            .map_meta(|variable, _| match variable {
+                "symbol" => Some(config.symbol),
+                _ => None,
+            })
+            .map_style(|variable| match variable {
+                "style" => Some(Ok(config.style)),
+                _ => None,
+            })
+            .map(|variable| match variable {
+                "subscription" => Some(Ok(subscription_name.as_ref().unwrap())),
+                _ => None,
+            })
+            .parse(None, Some(context))
+    });
+
+    module.set_segments(match parsed {
+        Ok(segments) => segments,
+        Err(error) => {
+            log::info!("Error in module `azure`:\n{}", error);
```

Comment:
This is supposed to be the same across all modules:
```suggestion
            log::error!("Error in module `azure`:\n{}", error);
```

Reply:
Thanks for the quick turn around. Should be fixed now.